rxbinding/src/main/java/com/jakewharton/rxbinding/widget/RxSearchView.java

```diff
@@ -0,0 +1,81 @@
+package com.jakewharton.rxbinding.widget;
+
+import android.widget.SearchView;
+
+import rx.Observable;
+import rx.functions.Action1;
+import rx.functions.Func1;
+
+import static com.jakewharton.rxbinding.internal.Preconditions.checkNotNull;
+
+/**
+ * Static factory methods for creating {@linkplain Observable observables} and {@linkplain Action1
+ * actions} for {@link SearchView}.
+ */
+public class RxSearchView {
+    /**
+     * Create an observable of {@linkplain SearchViewQueryTextEvent query text events}
+     * on {@code view}.
+     * <p>
+     * <em>Warning:</em> The created observable keeps a strong reference to {@code view}.
+     * Unsubscribe to free this reference.
+     */
+    public static Observable<SearchViewQueryTextEvent> queryTextEvents(SearchView view) {
+        checkNotNull(view, "view == null");
+        return Observable.create(new SearchViewQueryTextEventsOnSubscribe(view));
+    }
+
+    /**
+     * Create an observable of character sequences for query text changes on {@code view}.
+     * <p>
+     * <em>Warning:</em> The created observable keeps a strong reference to {@code view}.
+     * Unsubscribe to free this reference.
+     */
+
+    public static Observable<? extends CharSequence> queryTextChanges(SearchView view) {
+        checkNotNull(view, "view == null");
+        return RxSearchView.queryTextEvents(view)
+                .map(new Func1<SearchViewQueryTextEvent, CharSequence>() {
```

Comment:
Doing this is extremely inefficient. You'll notice all other operators which provide access to the "raw" data duplicate the OnSubscribe class to avoid the overhead of allocating the ViewEvent object just to throw it away.